### ISSUE TITLE
Document font size units in theme_bw() etc

### ISSUE THREAD

**Issue by hadley** [tidyverse/ggplot2#3458]
I assume they're points?

**Comment by clauswilke** [comment]
Yes. From the documentation of `gpar()`:

>  `fontsize` The size of text (in points)

Also:
``` r
library(grid)

g <- textGrob("pQgj", gp = gpar(fontfamily = "Helvetica", fontsize = 11))
convertHeight(grobHeight(g)+grobDescent(g), "pt")
#> [1] 10.3619543457031pt

g <- textGrob("pQgj", gp = gpar(fontfamily = "Times", fontsize = 11))
convertHeight(grobHeight(g)+grobDescent(g), "pt")
#> [1] 9.72039733886718pt

g <- textGrob("pQgj", gp = gpar(fontfamily = "Palatino", fontsize = 11))
convertHeight(grobHeight(g)+grobDescent(g), "pt")
#> [1] 10.7123846435547pt
```

<sup>Created on 2019-07-26 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by hadley** [comment]
Should be explicit that it's not `mm` unlike `geom_text()` etc, and should probably mention `.pt`

**Comment by clauswilke** [comment]
This has been improved somewhat with PR #3787 but there's probably room for further improvement, so leaving this open.

**Comment by boa50** [comment]
What can I do to help document this?

**Comment by teunbrand** [comment]
I'm considering this closed per #3787, #5260 and #5446.
